asapdiscovery-data/asapdiscovery/data/postera/manifold_data_validation.py

```diff
@@ -0,0 +1,66 @@
+from enum import Enum
+
+import pandas as pd
+
+
+class ManifoldAllowedColumns(Enum):
+    """
+    Enum of allowed columns for the P5 comp-chem team to update in postera.
+    """
+
+    SMILES = "SMILES"
```

Comment:
Do we actually refer to these columns via `ManifoldAllowedColumns.ASAP_VC_ID`, etc? If not, does it make sense to hard-code these ads defined values?

Alternatively, would a YAML file work for this, where instead of programmatically coding this, we have a flexible data-driven encoding of canonical names (`ASAP_VC_ID`) and SD tags (`asap_vc_id`), though I confess I'm not sure which name is supposed to be the SD tag and which is supposed to be the Python variable name that is extracted.

From a general philosophy standpoint, it's helpful to think about this as aiming to produce code that is reusable for other projects when possible, so that avoiding hard-coding project-specific choices in code that could instead be flexibly defined in configuration files (such as `project.yaml`) will make it easier for others to re-use the code.

Finally, I think we still need to align on SD tag names that parallel the CDD Vault protocol name structure, as well as are unique, relatively stable, extensible, and obvious to the chemists. For example, the chemists may not know what `chemgauss4` or `GAT` is. We may also want to use standard virus nomenclature---there is no `SARS2` or `MERS`, but `SARS-CoV-2`, `MERS-CoV`, `DENV`, `ZIKV`, `WNV`, etc. The target name may also be relevant. We're working on standardizing those target names [here](https://asapdiscovery.slack.com/archives/C03GYK0PHB4/p1684423929023879).

Reply:
Thanks John! should be easy enough to change to YAML files, I'll raise an issue while we continue to work on the SD tag names themselves.

Reply:
also it should be noted here that we're not dealing with `ASAP_VC_ID`s, but instead with `UUID_POSTERA_API`s. `ASAP_VC_ID`s don't get assigned until designs are submitted to a `design set` in the postera portal, and that currently happens after any computational modelling has been done. If this seems confusing (it is) I'm happy to explain further.